tests/components/image_processing/test_microsoft_face_detect.py

```diff
@@ -8,15 +8,16 @@
 import homeassistant.components.microsoft_face as mf
 
 from tests.common import (
-    get_test_home_assistant, assert_setup_component, load_fixture, mock_coro)
+    get_test_home_assistant_http, assert_setup_component, load_fixture,
+    mock_coro)
 
 
 class TestMicrosoftFaceDetectSetup(object):
     """Test class for image processing."""
 
     def setup_method(self):
         """Setup things to be run when tests are started."""
-        self.hass = get_test_home_assistant()
+        self.hass = get_test_home_assistant_http()
```

Comment:
So I've started down this road, and I'm starting to think it's not worth the distinction. Some of these aren't using network functions, but they do use other portions initialized by the http component, such as the base_url or the iframe test uses the FINGERPRINTS, etc.

I think even if we got it right here, it would be more confusing when people implement future tests, and would probably end up messed up.